### PR TITLE
First post for learning

### PR DESCRIPTION
**Pull request by Ender0Kai**
Will need to be actually updated.

### CODE REVIEW

**Comment by is4ac** [comment]
@Ender0Kai  Sweet sweet, is this ready for merging or are you still playing around with it?

**Comment by Ender0Kai** [comment]
Sure! Its a go.I mean, why not start somewhere. XD I’ll write something heartfelt today and get it there.

File: umwdevlogs/F1stP0st.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+publishDate: 2022-08-05
+excerpt: In case you were curious
+author: Ender
+draft: false
+tags: [Meaningless, Existensialism, art,]
+---
+
+# New art
+
+Check out this great fact I learned on youtube!
```
Comment (is4ac) [inline umwdevlogs/F1stP0st.mdx]: Nitpick: should we care about proper editing for these posts? e.g. capitalizing YouTube?

File: umwdevlogs/F1stP0st.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+tags: [Meaningless, Existensialism, art,]
+---
+
+# New art
+
+Check out this great fact I learned on youtube! 
+
+<img src="/images/Knitting is a science.png" width="1024" height="712" />
+
+This means something deeper I don't intend on sharing yet! but there you have it!
```
Comment (is4ac) [inline umwdevlogs/F1stP0st.mdx]: Nitpick: same question -- should we capitalize "But"?

**Review by is4ac — CHANGES_REQUESTED** [review]
Some minor grammatical things that we don't have to change, unless we want to hold our dev-blag posts to a high standard XD

File: umwdevlogs/F1stP0st.mdx (hunk 1 above)
Comment (Ender0Kai) [reply umwdevlogs/F1stP0st.mdx]: Aight. did a redo thingy mabob

File: umwdevlogs/F1stP0st.mdx (hunk 2 above)
Comment (Ender0Kai) [reply umwdevlogs/F1stP0st.mdx]: your face

**Review by Ender0Kai — COMMENTED** [review]
Uhhh... fixed the things?

**Review by is4ac — APPROVED** [review]
Looks great!

**Pull request merged**